targets/PhpSdk/source/PlayFabHttp.php.ejs

```diff
@@ -5,8 +5,9 @@ if (!class_exists("PlayFabSettings")) {
     {
         public static $versionString = "PhpSdk-<%- sdkVersion %>";
         public static $requestGetParams = array("sdk" => "<%- sdkVersion %>");
-        public static $PlayFabApiUrl = "https://{titleId}.playfabapi.com";
+        public static $productionEnvironmentUrl = ".playfabapi.com"; // This is only for customers running a private cluster.  Generally you shouldn't touch this
         public static $enableCompression = false; // Can't get this to work
+        public static $verticalName = <%- getVerticalNameDefault() %>; // The name of a customer vertical. This is only for customers running a private cluster. Generally you shouldn't touch this
         public static $titleId = null;
```

Comment:
Which... apparently means I'm a dumb-something something, because I made the same mistake.